calyx/src/cmdline.rs

```diff
@@ -23,8 +23,8 @@ pub struct Opts {
     pub file: Option<PathBuf>,
 
     /// Path to the primitives library.
-    #[structopt(long, short, required_unless = "list-passes")]
-    pub libraries: Vec<PathBuf>,
+    #[structopt(long, short)]
+    pub lib_path: Option<PathBuf>,
```

Comment:
If possible, make it so that when this is not specified, it defaults to the current directory.

Reply:
:+1: You should make it just be a `PathBuf` and then use the `default_value` flag to set the default value to the current directory

Reply:
I have a question.. can we put a filePath as a default value? I am trying to use std::env::current_dir() to get the current directory and use that as default and the compiler yells at me for excepecting a string reference...

Reply:
I don’t think you can put a pathbuf there. Putting the string “.” Should work.